app/controllers/concerns/hyrax/works_controller_behavior.rb

```diff
@@ -49,9 +49,9 @@ def new
 
       # admin_set_id is required on the client, otherwise simple_form renders a blank option.
       # however it isn't a required field for someone to submit via json.
-      # Set the first admin_set they have access to.
+      # Set the default admin set if it exists; otherwise, set to first admin_set they have access to.
       admin_set = Hyrax::AdminSetService.new(self).search_results(:deposit).first
-      curation_concern.admin_set_id = admin_set && admin_set.id
+      curation_concern.admin_set_id = AdminSet::DEFAULT_ID || (admin_set && admin_set.id)
```

Comment:
Does a user always have access to deposit in the default admin set?  If not, then I think this might cause problems.

Reply:
Good catch.  I'll modify this to be sure the user can access.